src/app/users/edit-user/edit-user.component.scss

```diff
@@ -0,0 +1,3 @@
+.password-never-expires-wrapper {
+  padding: 17.5px 0 17.5px 0;
+}
```

Comment:
use bootstrap classes instead of this

Reply:
@karantakalkar 
This is done using the **exact** same way in `create-user.component.scss` . Are you sure you want to change it?

Reply:
It's fine.